service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseExtensionServiceTest.java

```diff
@@ -18,94 +18,79 @@
+            .extensionDateOther(OTHER_DATE)
             .caseExtensionTimeList(OTHER_EXTENSION).build();
 
         LocalDate caseCompletionDate = service.getCaseCompletionDate(data);
 
-        assertThat(caseCompletionDate)
-            .isEqualTo(extensionDateOther);
+        assertThat(caseCompletionDate).isEqualTo(OTHER_DATE);
     }
 
     @Test
     void shouldGetCaseCompletionDateWhenSubmittingWith8WeekExtensionOther() {
-        LocalDate eightWeeksExtensionDateOther = LocalDate.of(2030, 11, 12);
-
         CaseData data = CaseData.builder()
             .caseExtensionTimeList(EIGHT_WEEK_EXTENSION)
             .caseExtensionTimeConfirmationList(OTHER_EXTENSION)
-            .eightWeeksExtensionDateOther(eightWeeksExtensionDateOther)
+            .eightWeeksExtensionDateOther(EXTENDED_OTHER)
             .build();
 
         LocalDate caseCompletionDate = service.getCaseCompletionDate(data);
 
-        assertThat(caseCompletionDate)
-            .isEqualTo(eightWeeksExtensionDateOther);
+        assertThat(caseCompletionDate).isEqualTo(EXTENDED_OTHER);
     }
 
     @Test
-    void shouldGetCaseCompletionDateWhenSubmittingWhenSubmittingWith8WeekExtension() {
-        LocalDate dateSubmitted = LocalDate.of(2030,11,11);
-
+    void shouldGetCaseCompletionDateSubmittingWith8WeekExtension() {
         CaseData data = CaseData.builder()
             .caseExtensionTimeList(EIGHT_WEEK_EXTENSION)
             .caseExtensionTimeConfirmationList(EIGHT_WEEK_EXTENSION)
-            .dateSubmitted(dateSubmitted)
+            .dateSubmitted(DATE_SUBMITTED)
             .build();
 
         LocalDate caseCompletionDate = service.getCaseCompletionDate(data);
 
-        assertThat(caseCompletionDate)
-            .isEqualTo(dateSubmitted.plusWeeks(8));
+        assertThat(caseCompletionDate).isEqualTo(EXTENDED_DATE);
     }
 
     @Test
     void shouldGetCaseCompletionDateFor8WeekExtension() {
-        CaseData data = CaseData.builder().dateSubmitted(LocalDate
-            .of(2020, 11, 20)).build();
+        CaseData data = CaseData.builder().dateSubmitted(DATE_SUBMITTED).build();
 
-        LocalDate expectedCaseCompletionDate = data.getDateSubmitted().plusWeeks(8);
+        LocalDate caseCompletionDate = service.getCaseShouldBeCompletedByDate(data).plusWeeks(8);
 
-        LocalDate caseCompletionDate = service.getCaseCompletionDateFor8WeekExtension(data);
-
-        assertThat(caseCompletionDate)
-            .isEqualTo(expectedCaseCompletionDate);
+        assertThat(caseCompletionDate).isEqualTo(EXTENDED_DATE);
     }
 
     @Test
     void shouldGetCaseSubmittedDateWhenNoCompletionDate() {
-        CaseData data = CaseData.builder().dateSubmitted(LocalDate
-            .of(2020, 11, 20))
-            .caseCompletionDate(LocalDate.of(2030, 11, 20)).build();
-
-        LocalDate expectedCaseCompletionDate = data.getCaseCompletionDate();
+        CaseData data = CaseData.builder().dateSubmitted(DATE_SUBMITTED).build();
```

Comment:
test name is now incorrect

Reply:
name was wrong before I think, now it looks fine to me - no caseCompletionDate, instead there is dateSubmitted